### DISCUSSION TITLE
Feature request: Control refresh rate

### DISCUSSION THREAD

**Discussion by schneems** [MonitorControl/MonitorControl#1350] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

My monitor flickers every time I connect it until I manually change my refresh rate from variable to 60hz. More details here https://www.reddit.com/r/Dell/comments/1221mz2/dell_s3221qs_monitor_flickers_when_connected_to_a/. 

It's tangential to setting brightness, but it would be amazing if it's possible to fix it somehow.

### Describe the solution you'd like

Set a default refresh rate for external monitors. I could use this to tell MonitorControl that I want external monitors to be at 60hz and on reconnect have it apply that setting for me automatically.

### Describe alternatives you've considered

Alternatively, a way to disable variable rate refresh. I don't know if that is exposed.

### Anything else?

- https://www.reddit.com/r/Dell/comments/1221mz2/dell_s3221qs_monitor_flickers_when_connected_to_a/
- https://www.reddit.com/r/MacOS/comments/qg9qki/updated_to_monterey_monitor_is_flickering_help/
- https://discussions.apple.com/thread/253306846
- https://www.howtogeek.com/805459/mac-flickering-external-screen/

**Comment by waydabber** [comment]
BetterDisplay has this feature (Config Protection): https://github.com/waydabber/BetterDisplay#readme
